rascal-vscode-extension/src/extension.ts

```diff
@@ -134,6 +118,12 @@ export function activateLanguageClient(context: vscode.ExtensionContext, languag
                 registerLanguage(context, parametricServer, lang);
             });
         }
+
+        let schemesReply:Promise<string[]> = client.sendRequest("rascal/filesystem/schemes");
+
+        schemesReply.then( (schemes:string[]) => {
```

Comment:
nitpick: Th `:string[]` part can be dropped, as it's inferred from the type.

Reply:
sure. I needed it to write the code, but if it can be dropped, by all means.